packages/mco/components/discovered-application-wizard/footer.tsx

```diff
@@ -107,16 +113,10 @@ export const EnrollDiscoveredApplicationFooter: React.FC<EnrollDiscoveredApplica
     };
 
     const handleNext = async () => {
-      switch (stepName) {
-        case enrollDiscoveredApplicationStepNames[
-          EnrollDiscoveredApplicationSteps.Review
-        ]:
-          setRequestInProgress(true);
-          await onSubmit();
-          setRequestInProgress(false);
-          break;
-        default:
-          moveToNextStep();
+      if (isReviewStep) {
+        await onSubmit(setRequestInProgress);
```

Comment:
no need of `await` here...
```suggestion
       onSubmit(setRequestInProgress);
```

Reply:
yes not required, I am doing anything after onSubmit function call.